kale/pipeline/multiomics_trainer.py

```diff
@@ -0,0 +1,227 @@
+# =============================================================================
+# Author: Sina Tabakhi, sina.tabakhi@gmail.com
+# =============================================================================
+
+"""
+Construct a pipeline to run MOGONET architecture based on PyTorch Lightning.
```

Comment:
run ... architecture? sounds weird and also missing an article

Reply:
The term "architecture" has been replaced with "method."

Reply:
What did you mean by "missing an article"? Just to clarify, I have already cited the original paper in the code.